kitchen-inspec.gemspec

```diff
@@ -20,11 +20,12 @@ Gem::Specification.new do |spec|
   spec.bindir        = "exe"
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
+  spec.required_ruby_version = ">= 2.1.0"
   spec.add_dependency "inspec", ">=0.22.0", "<1.0.0"
   spec.add_dependency "test-kitchen", "~> 1.6"
   spec.add_development_dependency "countloc", "~> 0.4"
   spec.add_development_dependency "bundler", "~> 1.10"
-  spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "rake", "~> 11.0"
```

Comment:
We've moved to putting our dev deps in the Gemfile. I'm totally fine with that. I didn't want to change things to much in this PR since I wasn't sure how you folks like to do things, but I can make that change now that I know you're OK with it.